JAVASCRIPT/Session_2/Get_the_middle_character.js

```diff
@@ -0,0 +1,6 @@
+// Get the Middle Character
+// http://www.codewars.com/kata/get-the-middle-character
+
+function getMiddle(inputStr) {
+    return inputStr.length % 2 == 0 ? inputStr.substring(inputStr.length / 2 - 1, inputStr.length / 2 + 1) : inputStr[parseInt(inputStr.length / 2)]
```

Comment:
you have multiple declarations of inputStr.length.
In an ideal scenario create a var with inputStr.length and reuse it